src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs

```diff
@@ -2408,7 +2409,6 @@ public new abstract class Arm64 : ArmBase.Arm64
             ///   A64: LD1 { Vn.2D, Vn+1.2D, Vn+2.2D, Vn+3.2D }, [Xn]
             /// </summary>
             public static unsafe (Vector128<double> Value1, Vector128<double> Value2, Vector128<double> Value3, Vector128<double> Value4) LoadVector128x4(double* address) { throw new PlatformNotSupportedException(); }
-#endif
```

Comment:
you forgot to remove this:

https://github.com/dotnet/runtime/blob/ac149359907423b90513a8fa392b6977fd919a7c/src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs#L2109-L2115

Reply:
Ah!